### DISCUSSION TITLE
Meshtastic reference

### DISCUSSION THREAD

**Discussion by n0qbh** [meshtastic/Discussions#125] [Firmware]
Is there a good place to look for the new features each new revision provides?
For example, I discovered when I upgraded one of my RAK 4631 nodes to 2.5 it became an orphan as all the others are 2.4.
The old Meshtastic discourse was a better platform in that it was more informative - IMHO
Mike

**Comment by garthvh** [comment]
There are extensive release notes on GitHub with each firmware release.

**Reply by n0qbh** [reply]
Being sorta new to GitHub, could you be more specific where they are please?
Thanks!

**Reply by fifieldt** [reply]
https://github.com/meshtastic/firmware/releases

**Comment by n0qbh** [comment]
I found what I was looking for here regarding the change in encryption starting with 2.5.
https://meshtastic.org/docs/overview/encryption/
